features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that leaves the current branch intact and deletes the previous branch
```

Comment:
Feature: deleting the previous Git branch sets main as the new previous Git branch